test/stripe/invoice_test.rb

```diff
@@ -87,8 +87,9 @@ class InvoiceTest < Test::Unit::TestCase
         assert_requested :get, "#{Stripe.api_base}/v1/invoices/upcoming",
                          query: {
                            customer: "cus_123",
-                           :'subscription_items[][plan]' => "gold",
-                           :'subscription_items[][quantity]' =>  2,
+                           subscription_items: [
+                             { plan: "gold", "quantity": "2" },
```

Comment:
You have a syntax error here: `quantity` should not be wrapped in quotes.

Reply:
Oops, great catch. Fixed in 2bc471d.